server/task_executor_schema_update_ghost_drop_original_table.go

```diff
@@ -21,5 +22,6 @@ type SchemaUpdateGhostDropOriginalTableTaskExecutor struct {
 
 // RunOnce will run SchemaUpdateGhostDropOriginalTable task once.
 func (exec *SchemaUpdateGhostDropOriginalTableTaskExecutor) RunOnce(ctx context.Context, server *Server, task *api.Task) (terminated bool, result *api.TaskRunResultPayload, err error) {
+	time.Sleep(time.Second * 5)
```

Comment:
This seems arbitrary. What's the purpose?

Reply:
Duplicate of https://github.com/bytebase/bytebase/pull/1315#discussion_r874595128.